walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,144 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
 
 public final class FindMeetingQuery {
+  private final int AVAILABLE = 0;
```

Comment:
I decided to keep these constants inside the FindMeetingQuery file to not alter the file structure. But I can move them to another file if needed. I think these variables probably can be used in other files.

Reply:
Thanks for bringing this up! I think that would be ideal so they can be reused, but non-blocking.